core/src/main/java/org/hibernate/ogm/datastore/spi/DatastoreProvider.java

```diff
@@ -46,4 +47,17 @@
 	 * @return the schema definer type
 	 */
 	Class<? extends SchemaDefiner> getSchemaDefinerType();
+
+	/**
+	 * Whether this dialect allows emulation of transactions.
+	 *
+	 * When transaction emulation is used, transactions only demarcate a unit of work. The transaction emulation will
+	 * make sure that at commit time all required changes are flushed, but there are otherwise no true transaction,
+	 * in particular rollback, semantics.
+	 *
+	 * @return {@code true} if this dialect allows transaction emulation, {@code false} otherwise.
+	 *
+	 */
+	@Experimental("This contract might evolve into something which differentiates in more detail various transactional capabilities (see OGM-763)")
+	boolean allowsTransactionEmulation();
```

Comment:
I guess it's ok for now. I still would have preferred to return an object here which would give us better odds for evolving the thing in a compatible manner. But `@Experimental` gives us a free ride :)